adapters/InMobi/InMobiAdapter/GADMAdapterInMobiBannerAd.h

```diff
@@ -0,0 +1,17 @@
+//
+//  GADMAdapterInMobiBannerAd.h
+//  IMAdMobAdapter
+//
+//  Created by Bavirisetti.Dinesh on 01/09/22.
+//
+
+#import <GoogleMobileAds/GoogleMobileAds.h>
+#import <InMobiSDK/IMBanner.h>
+
+@interface GADMAdapterInMobiBannerAd : NSObject<GADMediationBannerAd, IMBannerDelegate>
+
+/// Initializes the banner ad renderer.
+- (nonnull instancetype)initWithPlacementIdentifier:(nonnull NSNumber *)placementIdentifier;
```

Comment:
I don't think we need this method. We can just get the placement ID from `adConfiguration` and pass it into the InMobi banner object.